src/jdk.jdwp.agent/share/native/libdt_filesocket/fileSocketTransport.c

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright (c) 2018, 2024 SAP SE. All rights reserved.
+ * Copyright (c) 2018, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
```

Comment:
I think no Oracle copyright is necessary here.

Reply:
This file is based on socketTransport.c, so I've added this Oracle copyright.